### PR TITLE
fix hmd preview image

### PR DESCRIPTION
**Pull request by davidkelly**
Should now be accurate to what is shown on monitor.

### CODE REVIEW

**Comment by davidkelly** [comment]
test this please

**Comment by davidkelly** [comment]
No need for test plan -- will be tested as part of spectator-camera branch tests.  Note we will want to make this more generic (in TextureCache) soon.

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1546/HighFidelity-Beta-PR10938-e2426c538eb4b7488464a99ed44a26ccf1dc43d3.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1546/HighFidelity-Beta-PR10938-e2426c538eb4b7488464a99ed44a26ccf1dc43d3.dmg

File: libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp [hunk 1]
```diff
@@ -514,8 +514,14 @@ void OpenGLDisplayPlugin::renderFromTexture(gpu::Batch& batch, const gpu::Textur
     batch.setPipeline(_presentPipeline);
     batch.draw(gpu::TRIANGLE_STRIP, 4);
     if (copyFbo) {
-        gpu::Vec4i rect {0, 0, scissor.z, scissor.w};
-        batch.blit(fbo, rect, copyFbo, rect);
+        gpu::Vec4i sourceRect(scissor.x, scissor.y, scissor.x + scissor.z, scissor.y + scissor.w);
```
Comment (zfox23) [inline libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: _ohhhhh..._

Comment (davidkelly) [inline libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: ^^^ yea - they are used both ways, where the z and w are width and height, or the coordinates of the opposite corner.  Not obvious.

**Pull request merged**